theseus_features/Cargo.toml

```diff
@@ -29,7 +29,7 @@ pmu_sample_stop = { path = "../applications/pmu_sample_stop", optional = true }
 ps = { path = "../applications/ps", optional = true }
 pwd = { path = "../applications/pwd", optional = true }
 rm = { path = "../applications/rm", optional = true }
-shell = { path = "../applications/shell", optional = true }
+shell = { package = "shell_2", path = "../applications/shell_2", optional = true }
```

Comment:
add a separate feature here, this removes the existing shell as an optional component.